cmd/fluxd/main.go

```diff
@@ -60,13 +60,30 @@ const (
 	defaultGitSyncTag     = "flux-sync"
 	defaultGitNotesRef    = "flux"
 	defaultGitSkipMessage = "\n\n[ci skip]"
+
+	RequireECR = "ecr"
+)
+
+var (
+	RequireValues = []string{RequireECR}
 )
 
 func optionalVar(fs *pflag.FlagSet, value ssh.OptionalValue, name, usage string) ssh.OptionalValue {
 	fs.Var(value, name, usage)
 	return value
 }
 
+type stringset []string
```

Comment:
Can we unify this with the `StringSet` defined in `warming.go`?

Reply:
Technically, yes. They have different methods, and not quite compatible representations, but (for example) I could move `StringSet` into a neutral package, add `Contains` for main.go to use.

Reply:
Sounds good. There is also a stringset in policies.go . It would be good to unify all of them. However, I understand you may not want to do it in this PR, so I will leave it up to you.